ballerina/http_constants.bal

```diff
@@ -30,8 +30,11 @@ const string HTTP_SCHEME = "http://";
 # Represents https protocol scheme
 const string HTTPS_SCHEME = "https://";
 
-# Constant for the default listener endpoint timeout
-const decimal DEFAULT_LISTENER_TIMEOUT = 120; //2 mins
+# Constant for the default listener endpoint timeout in seconds
+const decimal DEFAULT_LISTENER_TIMEOUT = 60;
```

Comment:
Any reason to reduce the default listener timeout ?

Reply:
Decision was taken due to the current fast paced environments. 120 seconds of delayed response is quite high to expect a failure in an environment where the throughput is 20K. However this is configurable. Hence, depends on the requirement user can change it.